lms/templates/navigation.html

```diff
@@ -63,21 +83,54 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
     % endif
 
     % if user.is_authenticated():
-    <ol class="left nav-global authenticated">
+    <ol class="left ${main_nav_menu_classlist} authenticated">
       <%block name="navigation_global_links_authenticated">
         % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
-          <li class="nav-global-01">
+          <li class="item nav-global-01">
             <a href="${marketing_link('COURSES')}">${_('Find Courses')}</a>
           </li>
         % endif
         %if settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff:
-          <li>
+          <li class="item">
             ## Translators: This is short for "System administration".
             <a href="${reverse('sysadmin')}">${_("Sysadmin")}</a>
           </li>
         %endif
       </%block>
     </ol>
+    % if uses_pattern_library:
+    <div class="wrapper-user-menu dropdown-menu-container logged-in js-header-user-menu">
+      <a href="${reverse('dashboard')}" class="menu-title">
+        <span class="sr-only">${_("Dashboard for:")}</span>
+        <%
+          username = user.username
+          profile_image_url = get_profile_image_urls_for_user(user)['medium']
+        %>
+        <img class="menu-image" src="${profile_image_url}" alt="">
+        ${username}
+      </a>
+      <button type="button" class="menu-button button-more has-dropdown js-dropdown-button default-icon" aria-haspopup="true" aria-expanded="false" aria-controls="${_("Usermenu")}">
+        <span class="icon-fallback icon-fallback-img">
+          <span class="icon icon-angle-down" aria-hidden="true"></span>
+          <span class="text">${_("Usermenu dropdown")}</span>
+        </span>
+      </button>
+      <ul class="dropdown-menu list-divided is-hidden" id="${_("Usermenu")}" tabindex="-1">
+        <%block name="navigation_dropdown_menu_links" >
+        <%
+        if uses_pattern_library:
```

Comment:
I don't understand this ``if`` as it all seems to be inside a wider if clause that only applies to the pattern library.